### ISSUE TITLE
MeshCentral Router - Files Corrupted During Transfer

### ISSUE THREAD

**Issue by chrislockwood** [Ylianst/MeshCentralRouter#87]
This issue occurs when transferring files via MeshCentral Router - It is frequently observed that resulting files received by the destination are often corrupted, in the case of the example below the resulting file is often larger that its source.  

Sometimes this same file will transfer correctly, however that becomes less likely the larger the file, in contrast transferring via the web interface is flawless every time (albeit significantly slower).

**To Reproduce**
1. Launch MeshCentral Router and login to your server
2. Select a suitable target and followed by `<right click> -> Remote Files`
3. Navigate to your source and destination locations
4. Select a file (larger the better) and initiate the transfer.

**Expected behavior**
The copied file should match the same file size and hash of the original, note sometimes the file size may be correct while the hash is not identical.

**Screenshots**
https://user-images.githubusercontent.com/1902573/185762685-4aeff327-be37-4fa8-8c59-dd8f28e8805c.mp4

**Server Software (please complete the following information):**
 - OS: Ubuntu 22.04 LTS
 - Virtualization: Proxmox LCX
 - Network: LAN only
 - Version: 1.0.62
 - Node: 12.22.9

**Client Device (please complete the following information):**
 - Device: VM (same host as MeshCentral)
 - OS: Windows 10 Pro (21H2)
 - Network: Local (including the host)
 - Browser: Edge
 - MeshCentralRouter Version: 1.8.8231

**Remote Device (please complete the following information):**
 - Device: Lenovo Tiny (Desktop)
 - OS: Windows 10 Pro (21H2)
 - Network: Local
 - Current Core Version (if known): Mar 25 2022, 1659876711

**Additional context**
This issue was first discovered when remotely transferring files, in order to rule out VPN tunnels and firewall appliances I have recreated using a virtual machine located on the same physical network/host as the MeshCentral Server. 

Additional strangeness exists with port relay mapping for the purposes of VNC (this to a different legacy Win XP workstation) in that connections will drop without warning after several seconds, similar problems are experienced with port relay access for other protocols (and to other clients).  However that issue is out of scope of this ticket, I mention only because it may have some bearing on the problem reported here.

**Your config.json file**
```
{
  "$schema": "http://info.meshcentral.com/downloads/meshcentral-config-schema.json",
  "__comment1__": "This is a simple configuration file, all values and sections that start with underscore (_) are ignored. Edit a section and remove the _ in front of the
 name. Refer to the user's guide for details.",
  "__comment2__": "See node_modules/meshcentral/sample-config-advanced.json for a more advanced example.",
  "settings": {
    "_cert": "myserver.mydomain.com",
    "_WANonly": true,
    "LANonly": true,
    "MpsPort": 4433,
    "WebRTC": true,
    "_sessionKey": "MyReallySecretPassword1",
    "_port": 443,
    "_aliasPort": 443,
    "_redirPort": 80,
    "_redirAliasPort": 80
  },
  "domains": {
    "": {
      "_title": "MyServer",
      "_title2": "Servername",
      "_minify": true,
      "_newAccounts": true,
      "_userNameIsEmail": true
    }
  },
  "_letsencrypt": {
    "__comment__": "Requires NodeJS 8.x or better, Go to https://letsdebug.net/ first before trying Let's Encrypt.",
    "email": "myemail@mydomain.com",
    "names": "myserver.mydomain.com",
    "production": false
  }
}
```

**Comment by si458** [comment]
plz download the latest version from https://meshcentral.com/downloads.html
or
download it from meshcentral 1.1.24
and try again and re-open if issue still persists

**Comment by jwhips** [comment]
This issue persists for me.  
I'm copying a Fedora image of ~775mb using router and it corrupts the file (tried three times now).  I used 7-zip on both ends to product a sha-256 value that differs.  
I did confirm from the Fedora website that I do have a valid sha value; the download is not corrupt.

running version 1.1.24
Let me know if I can provide anything else.  My config file is very basic.
Thanks for all you do.
